lib/src/vDraw_basic.cpp

```diff
@@ -326,7 +213,7 @@ std::string imuDraw::getEventType()
 
 void imuDraw::draw(cv::Mat &image, const ev::vQueue &eSet, int vTime)
 {
-    auto radius = 20;
```

Comment:
@ghoshsuman - also note that this change is for the IMU and shouldn't be part of this pull request either. It was probably a mistake to change it but we should catch it in the code review